wfe/wfe.go

```diff
@@ -1618,3 +1620,52 @@ func uniqueLowerNames(names []string) []string {
 	sort.Strings(unique)
 	return unique
 }
+
+func (wfe *WebFrontEndImpl) RevokeCert(
```

Comment:
I think the `RevokeCert` function deserves a comment that explains that it is presently implementing only **one** method of ACME revocation: Signing the revocation request by signing it with the certificate to be revoked's private key and embedding the certificate to be revoked's public key as a JWK in the JWS.

Reply:
It might also be a good idea to emphasize that Pebble's idea of "revocation" is to forget the certificate exists. There might be the expectation that the revocation request once processed would percolate to a CRL or an OCSP response and that's not the case for Pebble.